prow/entrypoint/run.go

```diff
@@ -83,10 +82,20 @@ func (o Options) Run() error {
 		commandErr = err
 	case <-time.After(timeout):
 		logrus.Errorf("Process did not finish before %s timeout", timeout)
-		if err := command.Process.Kill(); err != nil {
-			logrus.WithError(err).Error("Could not kill process after timeout")
+		if err := command.Process.Signal(os.Interrupt); err != nil {
+			logrus.WithError(err).Error("Could not interrupt process after timeout")
+		}
+		gracePeriod := time.Duration(optionOrDefault(o.GracePeriodSeconds, DefaultGracePeriodSeconds)) * time.Second
```

Comment:
So there isn't a way for me to disable the grace period?

Reply:
Well if gracePeriod is zero we can just send SIGKILL immediately (either right after interrupt or else just skip the interrupt altogether)

Reply:
In what case would you not want a grace period?